app/views/pages/api_release_notes.html.haml

```diff
@@ -300,12 +270,11 @@
         %li
           %a{ href: '/api/documentation?#!/api/documentation#!/api/postApiExternalUsersClaimsAdvocatesHardshipValidate', target: '_blank' } advocate hardship claim validate
       %br/
-      %br/
       A list of valid Case Stages (to be used when creating a hardship claim) can be retrieved via:
-      = govuk_table do
-        = govuk_table_row do
-          = govuk_table_td('data-label': "Api") do
-            = 'GET /api/case_stages'
+
+      %ul.govuk-list.govuk-list--bullet
+        %li
+          GET /api/case_stages
```

Comment:
Hi @mpw5, just checking as it looks very different from everything else.

Reply:
Ah, good spot. I should have put something in the commit message to explain this.

I took a slightly different approach here (and a few other places in this file) where a table was being used just to show a single line. Rather than create a new table, which felt a bit redundant, I just used a bullet point instead.

This is what it looked like before:

<img width="808" alt="image" src="https://github.com/user-attachments/assets/653bf75c-14b8-4e32-b94b-16cd182c1f1a" />

and this is what it looks like now:

<img width="762" alt="image" src="https://github.com/user-attachments/assets/700031a3-17df-4c09-8303-0c4b4a8092d2" />

Reply:
Oh that makes sense - thanks @mpw5 :)